<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MSAExportSettings</class>
 <widget class="QWidget" name="MSAExportSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Whole alignment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom region</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectRegionButton">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="exportSeqNames">
     <property name="text">
      <string>Include sequences' names</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exportConsensus">
     <property name="text">
      <string>Include consensus</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="exportRuler">
     <property name="text">
      <string>Include ruler</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
